examples/process-persistence/README.md

```diff
@@ -34,21 +34,168 @@ example could use `h2`, `postgresql` or `mssql`. In container mode it can use `p
+```
+
+Let's take a look at each profile and the configurations in detail.
+
+### H2
+
+This is the maven profile for H2
+
+```
+<profile>
+    <id>h2</id>
+    <activation>
+        <activeByDefault>true</activeByDefault>
+    </activation>
+    <properties>
+        <quarkus.profile>h2</quarkus.profile>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jdbc-h2</artifactId>
+        </dependency>
+    </dependencies>
+</profile>
+```
+So H2 is the default database if we don't specify any profile. The only dependency needed is h2 jdbc driver.
+
+This is the configuration for H2
+```
+%h2.quarkus.datasource.db-kind=h2
+%h2.quarkus.datasource.devservices.properties.NON_KEYWORDS=VALUE,KEY
+```
+
+`quarkus.datasource.devservices.properties.NON_KEYWORDS=VALUE,KEY` is a generic property that is usually added 
+to the database connection url. The flyway scripts defines tables with columns names like `key` and `value`. But 
+these are reserved words when working with H2. So this property effectively allows to create columns with these 
+names without any problems.
+
+The H2 database runs in-process.
+
+### Postgresql
+
+This is the maven profile for Postgresql
+```
+<profile>
+    <id>postgresql</id>
+    <properties>
+        <quarkus.profile>postgresql</quarkus.profile>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jdbc-postgresql</artifactId>
+        </dependency>
+    </dependencies>
+</profile>
+```
+The only dependency needed is postgresql jdbc driver.
+
+This is the configuration for Postgresql
+```
+%postgresql.quarkus.datasource.db-kind=postgresql
+```
+
+The Postgresql database runs as a container. So make sure to install docker before running this example.
+
+### MS SQL Server
+
+This is the maven profile for MS SQL Server
+```
+<profile>
+    <id>mssql</id>
+    <properties>
+        <quarkus.profile>mssql</quarkus.profile>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jdbc-mssql</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.ibm.bamoe</groupId>
+            <artifactId>bamoe-mssql-mappings</artifactId>
+            <version>${version.com.ibm.bamoe}</version>
+        </dependency>
+    </dependencies>
+</profile>
+```
+The dependencies needed are mssql jdbc driver and `bamoe-mssql-mappings`.
+
+This is the configuration for MS SQL Server
+```
+%mssql.quarkus.datasource.db-kind=mssql
+%mssql.quarkus.hibernate-orm.mapping-files=META-INF/bamoe-data-index-orm.xml,META-INF/bamoe-job-service-orm.xml
+```
+
+The `bamoe-mssql-mappings` library provides mapping files required by hibernate. You could take a look at the library 
+[here](../../runtime-libraries/bamoe-database-support/bamoe-mssql-mappings/). This library is used in combination 
```

Comment:
@josedee I'd change this part a bit. Since this example will be delivered in a zip this link will be out of context. I think that this part should explain which are the mapping files, when should be used and the config property. Maybe you can copy that from the mappings README.md